radlab-ui/webapp/src/utils/terraform.ts

```diff
@@ -215,3 +216,62 @@ const checkModuleVariablesZero = async (moduleName: string) => {
     })
   return returnModuleVariableData
 }
+
+// To get Admin settings updated variables data
+const GCP_PROJECT_ID = envOrFail(
+  "NEXT_PUBLIC_GCP_PROJECT_ID",
+  process.env.NEXT_PUBLIC_GCP_PROJECT_ID,
+)
+
+export const getAdminSettingData = async () => {
+  const adminVariableData = await axios
+    .get(`/api/settings?projectId=${GCP_PROJECT_ID}`)
+    .then((res) => {
```

Comment:
Any reason you're mixing promises and async/await? Why not just make it all async/await?